qiskit_ibm_runtime/estimator.py

```diff
@@ -197,19 +197,35 @@ def __init__(
             skip_transpilation
         )
 
+        if isinstance(session, Session):
+            self._session = session
+        else:
+            backend = session or backend
+            self._session = get_default_session(service, backend)
+
+        backend_obj = self._session._service._backends.get(self._session._backend)
```

Comment:
```suggestion
        backend_obj = self._session.service.backend(self._session.backend())
```
Should avoid using private function/property when public ones are available. But it also means you'd need to check if the session actually has a backend.